go/master/client.go

```diff
@@ -15,16 +17,58 @@ type Addresser interface {
 // Client is the client of the master server.
 type Client struct {
 	conn *connection.Conn
+	ch   chan []byte
 }
 
 // NewClient creates a new Client.
 func NewClient(addr Addresser) *Client {
 	c := &Client{}
 	c.conn = connection.New()
+	c.ch = make(chan []byte)
 	go c.monitorMaster(addr)
+	go c.getRecords()
 	return c
 }
 
+func (c *Client) getRecords() {
+	for {
+		t, err := c.getTask()
+		if err != nil {
+			// TODO(helin): wait before move on with next
+			// getTask call.
+			log.Println(err)
```

Comment:
Can we all use `glog` instead of `log`, so we can do `glog.Errorf` or `glog.Infof` to separate info log and error log. Use `glog.V(10).Infof` to log more detailed debug logs